Firestore/core/test/firebase/firestore/util/async_queue_test.cc

```diff
@@ -38,6 +38,12 @@ const TimerId kTimerId3 = TimerId::WriteStreamConnectionBackoff;
 
 }  // namespace
 
+AsyncQueueTest::~AsyncQueueTest() {
+  if (queue) {
+    queue->EnqueueBlocking([] {});
```

Comment:
Optional: each time I see this pattern being used (and I think we use it quite a lot), I wonder if we should add a method like `AsyncQueue::Drain` or something.